src/controllers/game.controller.js

```diff
@@ -0,0 +1,74 @@
+import GameView from '../views/game.view';
+import PlayerController from './player.controller';
+import AppService from '../app.service';
+
+const GameController = () => {
+  const players = [];
+  const appService = AppService();
+  let gameInProgress = false;
+
+  // Game is finished if any board have all ships sinked.
```

Comment:
**Kindly remove commented code inside file, you can always use git history to access previous code right**:slightly_smiling_face: